cachi2/core/package_managers/bundler/lockfile-parser.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
```

Comment:
nitpick: I'd probably separate Python non-native resources to another subdirectory, e.g. `scripts`. Maybe I'd go as far as moving it out of the bundler package and centralize it somewhere under `cachi2/core` but that's just a tiny detail that can be tweaked later on, for now I'd just stuff this script into a subdirectory relative to the bundler package.